scroll.go

```diff
@@ -51,6 +52,15 @@ func NewScrollService(client *Client) *ScrollService {
 	return builder
 }
 
+// AddHeader sets headers on the request
+func (s *ScrollService) AddHeader(headerName string, headerValue string) *ScrollService {
```

Comment:
I have some comments here:

1. Can we rename this from `AddHeader` to `Header` please? All services are builders, and it looks really bad to have a builder pattern with:

```
service.
  AddHeader("...", "...").
  SetTimeout(...).
  SetXXX(...).
  SetYYY(...).
  Do(...)
```

instead of 

```
service.
  Header("...", "...").
  Timeout(...).
  XXX(...).
  YYY(...).
  Do(...)
```

2. Can we use variable names like `key` and `value` instead of `headerName` and `headerValue` (like [`http.Header#Add`](https://golang.org/pkg/net/http/#Header) does). It's clear that e.g. the `value` is the header value for a method called `Header`.